help.go

```diff
@@ -52,10 +52,10 @@ func renderHelp(fullName string, cmd Command, fl *pflag.FlagSet, w io.Writer) {
 
 	// If the command has aliases, add them to the output as a comma-separated list.
 	if spec.HasAliases() {
-		fmt.Fprintf(&b, "Aliases: %v", spec.Aliases)
+		fmt.Fprintf(&b, "Aliases: %s\n\n", strings.Join(spec.Aliases, ", "))
 	}
 	// Render usage and description.
-	usageAndDesc := fmt.Sprintf("%s%s\n", b.String(), spec.Desc)
+	usageAndDesc := fmt.Sprintf("%sDescription: %s\n", b.String(), spec.Desc)
```

Comment:
note that `%s` will automatically call `.String()`, so this can be simplified to pass only `b` instead of `b.String()`

Reply:
When doing `fmt.Fprintf(w, "%sDescription: %s\n", b, spec.Desc)` :

```
./help.go:58:2: Fprintf format %s has arg b of wrong type strings.Builder
FAIL    go.coder.com/cli [build failed]
```

It seems like we have to be explicit when using a `strings.Builder`

Reply:
TIL :) I'll need to dig into this to see what's going on.